core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -144,6 +144,13 @@ public static long getLong(String property, JsonNode node) {
     return pNode.asLong();
   }
 
+  public static Boolean getBoolOrNull(String property, JsonNode node) {
```

Comment:
you might want to add a small test for this one in `TestJsonUtil`

Reply:
Good call